src/style.css

```diff
@@ -0,0 +1,124 @@
+/* #region body */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+/* #endregion */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  content: "";
+  background-image: url(./images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+}
+
+.card__title {
+  font-weight: 500;
+  margin-bottom: 4px;
+}
+
+.card__subtitle {
+  display: flex;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__list {
+  display: flex;
+  list-style: none;
+  order: 0;
+}
+
+.card__star {
+  content: "";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  height: 16px;
+  width: 16px;
+}
+
+.card__star:last-child {
+  background-image: url(./images/star.svg);
+}
+
+.card__star:not(:last-child) {
+  margin-right: 4px;
```

Comment:
need to transfer all styles for stars to keep the ability to change the number of active stars by changing the modifier

Reply:
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.